scripts/config.py

```diff
@@ -589,33 +320,22 @@ def __init__(self, filename=None):
         """Read the Mbed TLS configuration file."""
 
         super().__init__()
-        self.configfile = MbedTLSConfigFile(filename)
-        self.settings.update({name: Setting(active, name, value, section, self.configfile)
+        configfile = MbedTLSConfigFile(filename)
+        self.configfiles.append(configfile)
```

Comment:
In config.py in development HEAD, I can see that Config, MbedTLSConfig and CryptoConfig reference only one configuration file (field configfile). I do not understand why because we move some classes to the framework now the classes mentioned above have a "configfiles" field.